nexus/src/nexus.rs

```diff
@@ -2479,8 +2495,11 @@ impl Nexus {
 
     pub async fn racks_list(
         &self,
+        opctx: &OpContext,
         pagparams: &DataPageParams<'_, Uuid>,
     ) -> ListResult<db::model::Rack> {
+        opctx.authorize(authz::Action::Read, &authz::FLEET).await?;
```

Comment:
Racks are faked-up -- not in the database -- so their access check is right in nexus.rs.